Dangerfile

```diff
@@ -32,6 +32,6 @@ end
 # Show Danger test coverage report from Kover for template
 # Report coverage of modified files, warn if total project coverage is under 80%
 # or if any modified file's coverage is under 90%
-kover_file = "template/build/reports/kover/merged/xml/report.xml"
+kover_file = "template-xml/build/reports/kover/merged/xml/report.xml"
 markdown "## Kover report:"
-shroud.reportKover "Template Unit Tests", kover_file, 80, 95, false
+shroud.reportKover "Template - XML Unit Tests", kover_file, 80, 95, false
```

Comment:
The same question as the workflow file. How about the `template-compose` in the future? Do we have the Kover for it too? If we change the path here, what will happen when we would like to do it for `template-compose`? 🤔

Reply:
@lydiasama We have used the `template` as the default before but once we rename it, we need to rename the path for this as well `template` -> `template-xml`; if not, the `CI` will be `failed for this time`, I guess once we already implemented [#333](https://github.com/nimblehq/android-templates/issues/333) we might have some task to filter template between `template-compose` and `template-xml` before we start work on [#332](https://github.com/nimblehq/android-templates/issues/332) If you have a suggestion, please let me know cc @Tuubz 🙇‍♀️

Reply:
@lydiasama Same as the other workflows, we'll add one more step for `template-compose` in #333  🙏

Reply:
FYI: I will leave `TODO` to avoid confusion at this time 🚀